src/cli/columnar_databases.rs

```diff
@@ -0,0 +1,114 @@
+use crate::cli::analytics::do_analytics_query;
+use crate::cli::generic_error;
+use crate::cli::util::{cluster_identifiers_from, get_active_cluster};
+use crate::state::State;
+use log::debug;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, IntoPipelineData, PipelineData, ShellError, Signature, SyntaxShape, Value,
+};
+use std::sync::{Arc, Mutex};
+
+#[derive(Clone)]
+pub struct ColumnarDatabases {
+    state: Arc<Mutex<State>>,
+}
+
+impl ColumnarDatabases {
+    pub fn new(state: Arc<Mutex<State>>) -> Self {
+        Self { state }
+    }
+}
+
+impl Command for ColumnarDatabases {
+    fn name(&self) -> &str {
+        "columnar databases"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("columnar databases")
+            .switch("with-meta", "Includes related metadata in the result", None)
```

Comment:
Is this being used?

Reply:
It was not, good spot. Have removed now.